lib/oath/views/utils.dart

```diff
@@ -178,3 +179,11 @@ Future<void> addOathAccount(BuildContext context, WidgetRef ref,
     );
   }
 }
+
+Future<void> setManagePassword(BuildContext context, WidgetRef ref,
```

Comment:
I guess this name is from "Set or Manage Password", but it kind of looks like a setter. I think we should just name it `managePassword`, which matches the name of the dialog as well.

Reply:
will fix! 👍🏻